uber/config.py

```diff
@@ -145,7 +145,7 @@ def DEALER_REG_SOFT_CLOSED(self):
     @request_cached_property
     def DEALER_APPS(self):
         with sa.Session() as session:
-            return session.query(sa.Group).filter(sa.Group.tables > 0, sa.Group.cost > 0, sa.Group.status == self.UNAPPROVED).count()
+            return session.query(sa.Group).filter(sa.Group.is_dealer, sa.Group.cost > 0, sa.Group.status == self.UNAPPROVED).count()
```

Comment:
This particular line won't actually work. Here we're using the `Group` _class_, instead of a `Group` _instance_. The reason `Group.tables` works whereas `Group.is_dealer` doesn't, is that `is_dealer` is a plain ole python `@property` and `tables` is a SQLAlchemy column (which are designed to be used in a filter like this).

I've dug into the SQLAlchemy source in the past, and that is some true black magic voodoo!

We could probably construct a SQL query to match what `is_dealer` is doing internally, but for this I think `tables > 0` will work.